coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -336,8 +337,11 @@ def __call__(self, message: fedora_messaging.api.Message):
                                             '\n\t'.join(map(str, tuples)))
             if self.keytab_file:
                 with self.koji_client.multicall(strict=True) as m:
-                    for (tag, nvr) in tuples:
-                        m.tagBuild(tag=tag, build=nvr)
+                    tasks = [m.tagBuild(tag=tag, build=nvr)
+                                    for (tag, nvr) in tuples]
+                watch_tasks(self.koji_client,
+                            [task.result for task in tasks],
```

Comment:
Does this work? It roughly reads like `tasks` would be defined out of scope of this line (but I wouldn't be surprised if somehow `with` doesn't change the local scoping).

Reply:
it works in testing

Reply:
Huh, TIL that `with` doesn't change scoping.

Reply:
yeah if I find out otherwise tomorrow when I do more tests I'll report back here.